src/MediaDeviceHandler.ts

```diff
@@ -38,6 +39,8 @@ export enum MediaDeviceHandlerEvent {
 export default class MediaDeviceHandler extends EventEmitter {
     private static internalInstance;
 
+    private audioSettings: AudioSettings;
```

Comment:
I think it's a little weird to have this here - it shouldn't be a problem to pull from the settings every time

Reply:
Removed.